### PR TITLE
Update apt keyring installation

### PR DESCRIPTION
**Pull request by MaKraMc**
The plain google signing key for apt does not work anymore. It needs to be "deamord" using gpg.

### CODE REVIEW

**Comment by sftim** [comment]
/sig release

Is this the right change to make?

File: content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ For more information on version skews, see:
 2. Download the Google Cloud public signing key:
 
    ```shell
-   sudo curl -fsSLo /etc/apt/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
+   sudo curl -fsSL https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-archive-keyring.gpg
```
Comment (sftim) [inline content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:160]:
```suggestion
   curl -fsSL https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-archive-keyring.gpg
```

Comment (MaKraMc) [reply content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:160]: Yes, I just thought of this as well.

**Comment by sftim** [comment]
/hold

SIG Release: feel free to unhold once this has an appropriate tech review

**Review by saschagrunert — APPROVED** [review]
That's good from the SIG Release side. Thank you!

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm

**Review by neolit123 — COMMENTED** [review]
should this be backported to the website branches in skew?

**Pull request merged**